src/pybind/mgr/nfs/export/export_utils.py

```diff
@@ -0,0 +1,322 @@
+class GaneshaConfParser:
```

Comment:
do we have unit tests for the conf parser?

Reply:
> do we have unit tests for the conf parser?

There are no unit tests for it.

Reply:
@sebastian-philipp do you have a suggestion on a best-practice location for unit tests that could be written?